File: spotipieBack/src/main/java/com/spotipie/domain/Token.java
package com.spotipie.domain;

public class Token {
  private String accessToken;
  private String refreshToken;
  private int expiresIn;

  public Token(String accessToken, String refreshToken, int expiresIn) {
    this.accessToken = accessToken;
    this.refreshToken = refreshToken;
    this.expiresIn = expiresIn;
  }

  public String getAccessToken() {
    return this.accessToken;
  }

  public void setAccessToken(String accessToken) {
    this.accessToken = accessToken;
  }

  public String getRefreshToken() {
    return this.refreshToken;
  }

  public void setRefreshToken(String refreshToken) {
    this.refreshToken = refreshToken;
  }

  public int getExpiresIn() {
    return this.expiresIn;
  }

  public void setExpiresIn(int expiresIn) {
    this.expiresIn = expiresIn;
  }
}

File: spotipie/src/domain/Token.ts
export class Token {
	constructor(private accessToken: string, private refreshToken: string, private expiresIn: number ) {}

	public static of(accessToken: string, refreshToken: string, expiresIn: number): Token {
		return new Token(accessToken,refreshToken, expiresIn);
	}

	public getAccessToken(): string {
		return this.accessToken;
	} 
}